price-feeder/oracle/oracle.go

```diff
@@ -148,13 +148,16 @@ func (o *Oracle) GetPrices() map[string]sdk.Dec {
 	return prices
 }
 
-// SetPrices retrieve all the prices from our set of providers as determined
-// in the config, average them out, and update the oracle's current exchange
-// rates.
+// SetPrices retrieve all the prices and candles from our set of providers as
+// determined in the config. If candles are available, use TVWAP in order
+// to determine prices. If candles are not available, use the most recent prices
+// with VWAP. Warn the the user of any missing prices, and filter out any faulty
+// providers which do not report prices within 2𝜎 of the others.
```

Comment:
Nit: Possibly change verbs to "retrieves", "uses", "uses", "warns", "filters"

Reply:
agree